automation/run-tests.sh

```diff
@@ -227,8 +230,20 @@ function modprobe_ovs {
     lsmod | grep -q ^openvswitch || modprobe openvswitch || { echo 1>&2 "Please run 'modprobe openvswitch' as root"; exit 1; }
 }
 
+function remove_tempdir {
+   rm -rf "$NMSTATE_TEMPDIR"
+}
+
+function vdsm_tests {
+  trap remove_tempdir EXIT
+  git -C "$NMSTATE_TEMPDIR" clone --depth 1 https://gerrit.ovirt.org/vdsm
+  cd "$NMSTATE_TEMPDIR/vdsm"
```

Comment:
Is the `cd` necessary?

Reply:
Done

Reply:
It actually is necessary. Otherwise we would end up with wrong configuration for the vdsm func tests.